dev/docker-compose.yml

```diff
@@ -55,12 +55,12 @@ services:
     - authn-local:/run/authn-local
     - ../ci/ldap-certs:/ldap-certs:ro
     # TODO: authenticators_oidc/test has a dep on this
-    - ../ci/test_suites/authenticators_oidc/keycloak:/authn-oidc/keycloak/scripts
+    - ../ci/keycloak:/authn-oidc/keycloak/scripts
```

Comment:
didn't we move this to `ci/oauth/keycloak`?

Reply:
nice catch thanks